rlmolecule/alphazero/alphazero_problem.py

```diff
@@ -0,0 +1,33 @@
+from abc import abstractmethod
+
+from rlmolecule.alphazero.alphazero_vertex import AlphaZeroVertex
+from rlmolecule.mcts.mcts_problem import MCTSProblem
+
+
+class AlphaZeroProblem(MCTSProblem):
+
+    @abstractmethod
+    def get_value_estimate(self, parent: AlphaZeroVertex) -> (float, {AlphaZeroVertex: float}):
```

Comment:
I thought `policy` worked OK here, since it's getting the child priors in addition to the value estimate, plus I think that's how it was referred to in the AZ literature. We just might want to make sure that we reserve `policy` for this instance in the code?